JSDemos/Demos/Scheduler/AllDayPanelMode/React/App.tsx

```diff
@@ -12,14 +9,15 @@ const views: SchedulerTypes.Properties['views'] = [{
   name: '4 Days',
   intervalCount: 4,
 }, 'week'];
+
 const allDayPanelItems = ['all', 'allDay', 'hidden'];
 
 const App = () => {
-  const [allDayPanelMode, setAllDayPanelMode] = React.useState<SchedulerTypes.Properties['allDayPanelMode']>('allDay');
+  const [allDayPanelMode, setAllDayPanelMode] = React.useState<'all' | 'allDay' | 'hidden'>('allDay');
```

Comment:
If there's no correct type for `allDayPanelMode` imported from devextreme, it's better to set it as @ivanblinov2k17 wrote
We need to show how to use our types, not how to create new ones